src/AutoMapper.sln

```diff
@@ -34,6 +34,20 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "AutoMapper", "AutoMapper\Au
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoMapper.Net4", "AutoMapper.Net4\AutoMapper.Net4.csproj", "{8C955148-FA96-4F0D-880D-ABB6770F54E9}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoMapper.WinRT", "AutoMapper.WinRT\AutoMapper.WinRT.csproj", "{B80D950D-CC8D-4287-96B1-EEA037825E90}"

```

Comment:
These changes need to be reverted, you shouldn't have any changes to project files, solution files etc.